xarray/core/nanops.py

```diff
@@ -91,35 +91,18 @@ def nanargmin(a, axis=None):
     fill_value = dtypes.get_pos_infinity(a.dtype)
```

Comment:
Move this line inside the “if” statement if it’s only used in that branch.

Reply:
Sure, done.